docs/ui/preferences/virtual-machine/emulation.md

```diff
@@ -11,18 +11,18 @@ title: Emulation (macOS)
 
  ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation.png)
 
-- The "[QEMU](https://www.qemu.org/documentation/)" option is enabled by default and is used to run a guest operating system.
+The [**QEMU**](https://www.qemu.org/documentation/) option is enabled by default and is used to run a guest operating system. You can switch the virtual machine type after the first run.
```

Comment:
```suggestion
The [**QEMU**](https://www.qemu.org/documentation/) option is enabled by default and turns on a guest operating system. You can switch the virtual machine type after the first run.
```

I guess I'm quibbling over the option being used to run something (like electricity to a device) vs turning a feature on (like flipping a power switch)

Reply:
Just applied!